abb_experimental/package.xml

```diff
@@ -18,9 +18,12 @@
   <exec_depend>abb_irb120_moveit_config</exec_depend>
   <exec_depend>abb_irb120_support</exec_depend>
   <exec_depend>abb_irb120t_moveit_config</exec_depend>
+  <exec_depend>abb_irb1600_support</exec_depend>
   <exec_depend>abb_irb2600_support</exec_depend>
   <exec_depend>abb_irb4600_support</exec_depend>
   <exec_depend>abb_irb52_support</exec_depend>
+  <exec_depend>abb_irb6650s_support</exec_depend>
+  <exec_depend>abb_irb7600_support</exec_depend>
 
   <export>
```

Comment:
maybe add <architecture_independent/> ?

https://github.com/ros-infrastructure/rep/blob/240583e7c1b5148e0dc939bd40a628fda047aa26/rep-0140.rst#architecture_independent

"Specifying <architecture_independent/> is recommended for metapackages..."